modules/local/cellprofiler/illuminationcorrection/tests/main.nf.test

```diff
@@ -0,0 +1,77 @@
+// TODO nf-core: Once you have added the required tests, please run the following command to build this file:
+// nf-core modules test cellprofiler/illuminationcorrection
+nextflow_process {
+
+    name "Test Process CELLPROFILER_ILLUMINATIONCORRECTION"
+    script "../main.nf"
+    process "CELLPROFILER_ILLUMINATIONCORRECTION"
+
+    tag "modules"
+    tag "modules_"
+    tag "cellprofiler"
+    tag "cellprofiler/illuminationcorrection"
+
+    // TODO nf-core: Change the test name preferably indicating the test-data and file-format used
+    test("cellprofiler corrects illumination") {
+
+        // TODO nf-core: If you are created a test for a chained module
+        // (the module requires running more than one process to generate the required output)
+        // add the 'setup' method here.
+        // You can find more information about how to use a 'setup' method in the docs (https://nf-co.re/docs/contributing/modules#steps-for-creating-nf-test-for-chained-modules).
+
+        when {
+            process {
+                """
+                // TODO nf-core: define inputs of the process here. Example:
+                params.modules_testdata_base_path = "/Users/german/code/test-datasets/"
+
+                input[0] = [
+                    [ id:'BR00117035' ], // meta map
+                    file('${moduleDir}/tests/testdata/load_data.csv', checkIfExists: true),
+                ]
+                input[1] = file(params.modules_testdata_base_path + 'minimal_dataset/cpg0016-jump/source_4/images/2021_04_26_Batch1/images/BR00117035__2021-05-02T16_02_51-Measurement1/Images/', checkIfExists: true)
+                input[2] = file(params.modules_testdata_base_path + 'minimal_dataset/cpg0016-jump/source_4/workspace/pipelines/2021_04_26_Batch1/illum.cppipe', checkIfExists: true)
```

Comment:
Don't use `params.modules_testdata_base_path`. Instead provide direct urls to the raw files on github.